File: Assignment_22/Program1.c
/*Accept Character from user and check whether it is alphabet or not (A-Z a-z)
Input: F
Output: TRUE
*/

#include<stdio.h>

#define True 1
#define False 0

typedef int bool;

bool ChkAlpha(char ch)
{
    if((ch >= 'A' && ch <= 'Z') || (ch >= 'a' && ch <= 'z'))
    {
        return True;
    }
    else
    {
        return False;
    }
}

int main()
{
    char cValue = '\0';
    bool bRet = False;

    printf("Enter the character:");
    scanf("%c",&cValue);

    bRet = ChkAlpha(cValue);

    if(bRet == True)
    {
        printf("It is a character");
    }
    else
    {
        printf("It is not a character");
    }
    
    return 0;
}

File: Assignment_56/Program1.java
/* Write a java program which accepts two strings from user and check whether first string is the rotation of second string or not.
Input :   abcdefg  cdefgab
Output :  True
*/

import java.util.*;

class MyString
{
    public boolean Rotation(String str1,String str2)
    {
        if(str1.length() != str2.length())
        {
            return false;
        }

        String combine = str2 + str2;

        return combine.contains(str1);
    }
}

class Program1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter first string : ");
        String str1 = sobj.nextLine();

        System.out.println("Enter second string : ");
        String str2 = sobj.nextLine();

        MyString obj = new MyString();
        boolean bRet = obj.Rotation(str1,str2);

        if(bRet)
        {
            System.out.println("True");
        }
        else
        {
            System.out.println("False");
        }

        sobj.close();
    }
}